### PR TITLE
KFSTP-2014 Create core and web module

### PR DESCRIPTION
**Pull request by kkronenb**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,8 @@ sftp-config.json
 *.tmproject
 tmtags
 target/
+kfs-core/target
+kfs-web/target
```
Comment (bhutchinson) [inline .gitignore]: I wonder if we could simplify this with wildcards. something like */target?

Comment (kkronenb) [inline .gitignore]: probably, I was also wondering what we should do about /src/main/webapp*

Comment (bhutchinson) [inline .gitignore]: do those just become kfs-web/src/main/webapp*?

Comment (bhutchinson) [inline .gitignore]: Also, why provided scope on kfs-help in the main pom.xml?

Comment (kkronenb) [inline .gitignore]: the webapps ones are rice artifacts we want to avoid right?

Comment (kkronenb) [inline .gitignore]: scope provided on help is to keep it from being packaged in the war since it was already overlaid to static

**Pull request merged**